src/Transpiler.ts

```diff
@@ -1303,7 +1314,7 @@ export abstract class LuaTranspiler {
         }
     }
 
-    public transpileArrayCallExpression(node: ts.CallExpression): string {
+    public transpileArrayCallExpression(node: ts.CallExpression, noException?: boolean ): string {
```

Comment:
I really don't like this boolean hiding an exception. It is probably better to make a function `tsHelper.isDefaultArrayCallExpression(node: ts.CallExpression)` and just not call `transpileArrayCallExpression` if that returns false.